[2024-11-04T12:55:46.707+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-04T12:55:46.724+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: yandex_task_dag.view_creation manual__2024-11-04T12:38:18.871536+00:00 [queued]>
[2024-11-04T12:55:46.733+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: yandex_task_dag.view_creation manual__2024-11-04T12:38:18.871536+00:00 [queued]>
[2024-11-04T12:55:46.734+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-11-04T12:55:46.749+0000] {taskinstance.py:2888} INFO - Executing <Task(PostgresOperator): view_creation> on 2024-11-04 12:38:18.871536+00:00
[2024-11-04T12:55:46.757+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=334) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-04T12:55:46.758+0000] {standard_task_runner.py:72} INFO - Started process 336 to run task
[2024-11-04T12:55:46.759+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'yandex_task_dag', 'view_creation', 'manual__2024-11-04T12:38:18.871536+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpz9g2vpit']
[2024-11-04T12:55:46.760+0000] {standard_task_runner.py:105} INFO - Job 27: Subtask view_creation
[2024-11-04T12:55:46.815+0000] {task_command.py:467} INFO - Running <TaskInstance: yandex_task_dag.view_creation manual__2024-11-04T12:38:18.871536+00:00 [running]> on host 94d9180b5540
[2024-11-04T12:55:46.914+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='yandex_task_dag' AIRFLOW_CTX_TASK_ID='view_creation' AIRFLOW_CTX_EXECUTION_DATE='2024-11-04T12:38:18.871536+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-04T12:38:18.871536+00:00'
[2024-11-04T12:55:46.915+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-04T12:55:46.916+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-04T12:55:46.916+0000] {logging_mixin.py:190} INFO - Current task name:view_creation state:running start_date:2024-11-04 12:55:46.725305+00:00
[2024-11-04T12:55:46.917+0000] {logging_mixin.py:190} INFO - Dag name:yandex_task_dag and current dag run status:running
[2024-11-04T12:55:46.917+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-04T12:55:46.918+0000] {sql.py:266} INFO - Executing: 
[2024-11-04T12:55:46.927+0000] {base.py:84} INFO - Retrieving connection 'postgres'
[2024-11-04T12:55:46.932+0000] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/operators/sql.py", line 272, in execute
    output = hook.run(
             ^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 450, in run
    raise ValueError("List of SQL statements is empty")
ValueError: List of SQL statements is empty
[2024-11-04T12:55:46.945+0000] {logging_mixin.py:190} INFO - Task instance in failure state
[2024-11-04T12:55:46.946+0000] {logging_mixin.py:190} INFO - Task start:2024-11-04 12:55:46.725305+00:00 end:2024-11-04 12:55:46.944465+00:00 duration:0.21916
[2024-11-04T12:55:46.947+0000] {logging_mixin.py:190} INFO - Task:<Task(PostgresOperator): view_creation> dag:<DAG: yandex_task_dag> dagrun:<DagRun yandex_task_dag @ 2024-11-04 12:38:18.871536+00:00: manual__2024-11-04T12:38:18.871536+00:00, state:running, queued_at: 2024-11-04 12:55:41.524241+00:00. externally triggered: True>
[2024-11-04T12:55:46.948+0000] {logging_mixin.py:190} INFO - Failure caused by List of SQL statements is empty
[2024-11-04T12:55:46.948+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=yandex_task_dag, task_id=view_creation, run_id=manual__2024-11-04T12:38:18.871536+00:00, execution_date=20241104T123818, start_date=20241104T125546, end_date=20241104T125546
[2024-11-04T12:55:46.963+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-04T12:55:46.964+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 27 for task view_creation (List of SQL statements is empty; 336)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3004, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3158, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3182, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/operators/sql.py", line 272, in execute
    output = hook.run(
             ^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 450, in run
    raise ValueError("List of SQL statements is empty")
ValueError: List of SQL statements is empty
[2024-11-04T12:55:47.015+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-11-04T12:55:47.025+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/models/baseoperator.py:1378: AirflowProviderDeprecationWarning: Call to deprecated class PostgresOperator. (Please use `***.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
  result = cls.__new__(cls)

[2024-11-04T12:55:47.040+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-04T12:55:47.046+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
